addOns/spiderAjax/src/main/java/org/zaproxy/zap/extension/spiderAjax/AjaxSpiderTarget.java

```diff
@@ -137,31 +151,44 @@ public Target toTarget() {
      * @return a new builder, never {@code null}.
      */
     public static Builder newBuilder(Session session) {
-        return new Builder(session);
+        return new Builder(session, null);
+    }
+
+    /**
+     * Creates a new build of targets.
+     *
+     * @param extension the extension.
```

Comment:
These are both kinda awkward. Maybe "a new targets builder" and obviously the extension is an extension, maybe mention it's ExtensionAjax being passed in?

Reply:
I think the original should be `builder` instead of `build`, would that be better?

Re the parameters, there's not much to say, the type is already indicated in the parameter itself but can add that. fyi, I'm going to remove these methods at some point.

Reply:
Yeah "builder" does seem better. Don't worry about the params then 😉